dev-haskell/snap/snap-1.1.3.3.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# ebuild generated by hackport 0.9.0.0.9999
+#hackport: flags: -old-base
+
+CABAL_HACKAGE_REVISION=3
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
```

Comment:
If the test deps are commented out, then `test-suite` should be removed from `CABAL_FEATURES` as well. This is usually only necessary if the tests won't build.